icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -188,7 +183,7 @@ ant proddata 2>&1 | tee /tmp/cldr-newData-proddataLog.txt
 # build-icu-data.xml file, such as adding new locales etc.
 
 cd $TOOLS_ROOT/cldr/cldr-to-icu
-CLDR_DIR=$CLDR_TMP_DIR/production ant -f build-icu-data.xml | tee /tmp/cldr-newData-builddataLog.txt
+ant -f build-icu-data.xml -DcldrDataDir="$CLDR_TMP_DIR/production" | tee /tmp/cldr-newData-builddataLog.txt
```

Comment:
Given the new /tools/cldr/build.xml, should these instructions direct readers to that file instead of build-icu-data.xml, which is becoming more of an internal detail?

Reply:
Eventually. Given the new build file is very much work in progress and the existing things work fine and are tested, I'd leave the docs referencing the "old" way until we're thoroughly happy with things.

Reply:
I don't see why we are adding build.xml without tying it into the instructions.  It'll be a file with no purpose until someone (who?) comes around to document it later.

Reply:
It's not finished yet and the people who would be using it (Peter et al) haven't looked at it. I'd always do the docs integration steps in conjunction with them. Let's not make this more onerous than it needs to be right now.